server/services/sentence-plan/goalService.ts

```diff
@@ -17,6 +17,11 @@ export default class GoalService {
     return restClient.post<Goal>({ path: `/plans/${parentPlanUuid}/goals`, data: goal })
   }
 
+  async updateGoal(goal: NewGoal, parentPlanUuid: string) {
+    const restClient = await this.sentencePlanApiClient.restClient('Saving goal data')
+    return restClient.patch<Goal>({ path: `/plans/${parentPlanUuid}/goals`, data: goal })
```

Comment:
So the API endpoint we've created is at `/goals/goalUuid` can you update this to match please?

Reply:
> So the API endpoint we've created is at `/goals/goalUuid` can you update this to match please?

Done

Reply:
This commit https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/88/commits/ab1dc1501c6daae24c02e0b7e87e8db8e451549e modifies saveGoal, not updateGoal. saveGoal was previously correct, but the `path` for updateGoal should be `/goals/${goalUuid}`

Reply:
sorry my bad, I just did opposite. Resolved now

Reply:
The HTTP methods are now wrong for each of the calls, and the path for update is still wrong.

Reply:
hoping its all fine this time... really off track this morning

Reply:
So close: you've put `/goals/${goalUuid}/goals` but it should be `/goals/${goalUuid}`

Reply:
updated again

Reply:
Yay!